.env-poa-sokol.example

```diff
@@ -0,0 +1,15 @@
+SUBGRAPH_NAME=circlesubi/circlesubi-staging
+SUBGRAPH_NETWORK=poa-sokol
+
+GRAPH_ADMIN_NODE_ENDPOINT=https://api.thegraph.com/deploy/
```

Comment:
no seria sin deploy  o por que esta el deploy ahi?

Reply:
This is the endpoint we've been using. If it doesn't work we'll update it.

Reply:
This endpoint will be compatible with the self-hosted graph-node too.

Reply:
but I have seen this one in various  config files
<graph:
  # Graph node URL
  endpoint: https://api.thegraph.com without the `deploy`

Reply:
Can you link me to the files?

Reply:
https://github.com/CirclesUBI/circles-iac/blob/main/helm/circles-infra-suite/values.yaml#L61

Reply:
Note that here we need the admin api, not the public query api